src/pages/en/guides/content-collections.md

```diff
@@ -0,0 +1,480 @@
+  # All newsletters have the same frontmatter properties
+  newsletter/
+    week-1.md
+    week-2.md
+    week-3.md
+  # All blog posts have the same frontmatter properties
+  blog/
+    columbia.md
+    enterprise.md
+    endeavour.md
+```
+
+### Organizing with nested directories
+
+Collections are considered **one level deep**, so you cannot nest collections (or collection schemas) within other collections. However, we do allow nested directories to better organize your content. This is vital for certain use cases like internationalization:
+
+```bash
+src/content/
+  docs/
+    # docs schema applies to all nested directories 👇
+    en/
+    es/
+    ...
+```
+
+All nested directories will share the same schema defined for the top-level collection, if any (**docs** in this example).
+
+See [getting from nested directories](#getting-from-nested-directories) to see how folders are treated when retrieving collections.
+
+## Adding a schema
+
+Schemas are an optional way to enforce frontmatter types in a collection. To configure schemas, you can create a `src/content/config.{js|mjs|ts}` file. This file should:
+
+1. `export` a `collections` object, with each object key corresponding to the collection's folder name. We will offer a `defineCollection` utility [similar to the `defineConfig` helper](/en/guides/configuring-astro/#the-astro-config-file) (see example below).
+2. Use a [Zod object](https://github.com/colinhacks/zod#objects) to define schema properties. The `z` utility will be built-in and exposed by `astro:content`.
+
+For instance, say every `blog/` entry should have a `title`, `slug`, a list of `tags`, and an optional `image` url. We can specify each object property like so [using Zod functions](https://github.com/colinhacks/zod):
+
+```ts
+// src/content/config.ts
+import { z, defineCollection } from 'astro:content';
+
+const blog = defineCollection({
+  schema: {
+    title: z.string(),
+    slug: z.string(),
+    image: z.string().optional(),
+    tags: z.array(z.string()),
+  },
+});
+
+export const collections = { blog };
+```
+
+:::tip
+Browse the [**Zod quick reference**](#zod-quick-reference) for a rundown on the basics!
+:::
+
+You can also include dashes `-` in your collection name using a string as the key. For example, to configure the collection `src/content/my-newsletter`, you may do the following:
+
+```ts
+const myNewsletter = defineCollection({...});
+
+export const collections = { 'my-newsletter': myNewsletter };
+```
+
+### Why Zod?
+
+We chose [Zod](https://github.com/colinhacks/zod) since it offers key benefits over plain TypeScript types. Namely:
+- specifying default values for optional fields using `.default()`
+- checking the *shape* of string values with built-in regexes, like `.url()` for URLs and `.email()` for emails
+- transforming a frontmatter value into another value, like parsing a publish date to [a `Date` object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date).
```

Comment:
Ah, part of me is still in unsure of reception to Zod here. Since this is the first introduction of Zod in the project and in our docs, I felt a "why Zod" was worth the call-out. However, I noticed the quick reference coming after this instead of before felt awkward. We could change this to "Zod-specific features" and move it afterwards.